qcoro/CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ add_qcoro_library(
         coroutine.h
         macros_p.h
         waitoperationbase_p.h
+        qcorotask.h
```

Comment:
This shouldn't be needed, it should get automatically installed through the `CAMELCASE_HEADERS` (`QCoroTask` looks up `qcorotask.h` and both will be installed).